extensions/amp-analytics/0.1/test/test-vendors-new.js

```diff
@@ -104,6 +104,30 @@ describes.realWin(
               config = analytics.config_;
               done();
             });
+
+            // Have to get service after analytics element is created
+            const variableService = variableServiceForDoc(doc);
+
+            window.sandbox
+              .stub(variableService, 'getMacros')
+              .callsFake(function() {
+                // Add all the macros in amp-analytics
+                const elementMacros = {
```

Comment:
Can you store the elementMacros to `this.elementMacros_` maybe? Then we don't need to worry about updating the list here.